api/python/provisioner/config.py

```diff
@@ -583,6 +585,7 @@ class HashType(Enum):
 
     MD5 = "md5"
     SHA256 = "sha256"
+    SHA384 = "sha384"
```

Comment:
Why this odd one?
Ideally we should stick with the more secure **_sha512_** only.

Reply:
Ok. I set 512 as a default. But I am not sure that ts would be more secure in all the cases. Please check [that](https://en.wikipedia.org/wiki/Hash_function_security_summary), length extension attacks part..